client/my-sites/post-type-list/post-actions-ellipsis-menu/edit.jsx

```diff
@@ -32,6 +32,7 @@ function PostActionsEllipsisMenuEdit( { translate, canEdit, status, editUrl, bum
 			onClick={ bumpStat }
 			icon="pencil"
 			onMouseOver={ preloadEditor }
+			onFocus={ preloadEditor }
```

Comment:
Just nothing this seems unrelated to this PR (but a valid change!)

Reply:
The linter wasn't too happy about the lack of an `onFocus`, so I added it 🙂 It seemed like a safe enough improvement.